src/style.css

```diff
@@ -1 +1,68 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+* {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  font-style: normal;
+  color: #3d4e61;
+  box-sizing: border-box;
+}
+
+.search-bar {
+  margin-top: 20px;
```

Comment:
You've added a margin-top to the `.search-bar` class. It's better to be consistent with your margins. Add only top or bottom, but not both, to avoid potential margin collapse. (Refer to the CSS checklist)